src/main/scala/io/viash/functionality/resources/NextflowScript.scala

```diff
@@ -84,13 +100,15 @@ case class NextflowScript(
           |  main:
           |  output_ch = input_ch
           |    | preprocessInputs(config: config)
-          |    | main
+          |    | main_wf
```

Comment:
I had previously overlooked this, but the `main_w` should use the entrypoint of the NextflowScript